app/views/api/v2/job_invocations/base.json.rabl

```diff
@@ -15,3 +19,11 @@ end
 child :task => :dynflow_task do
   attributes :id, :state
 end
+
+node :permissions do |invocation|
```

Comment:
Could we hide this behind a parameter? Similarly to what katello does https://github.com/Katello/katello/blob/master/app/views/katello/api/v2/subscriptions/permissions.rabl#L3 ?